### DISCUSSION TITLE
Add pollingWaitTimeMs to UpdatableOptions

### DISCUSSION THREAD

**Discussion by maximpoleley** [bbc/sqs-consumer#434] [Feature Request]
### Background

Hi!

We currently have a need to be able to dynamically update pollingWaitTimeMs. 

Is it okay to add pollingWaitTimeMs to UpdatableOptions? If so, I'll prepare a pr.

Thank you!

### Objectives

pollingWaitTimeMs can be updated with updateOption function.

**Comment by nicholasgriffintn** [comment]
So this is possible, however, I would note that any timeouts that have already started wouldn't respect the new setting, so it may be possible that updating the setting without cancelling existing timeouts may confuse users, so it may be that we also want to do that when this option is updated.

Happy to take a PR for it.

**Reply by maximpoleley** [reply]
If Im understand correctly `pollingWaitTimeMs` being set only after current poll cycle is done. So when we do update `pollingWaitTimeMs` it will only affect next polling cycle without interrupting any ongoing process therefore it can be safely updated. Is that correct? 
https://github.com/bbc/sqs-consumer/blob/main/src/consumer.ts#L223

**Reply by nicholasgriffintn** [reply]
It would yeah, we'd probably just want to document that behaviour somewhere just so people know that's what will happen.

**Reply by maximpoleley** [reply]
Please take a look at the pr https://github.com/bbc/sqs-consumer/pull/436

**Reply by maximpoleley** [reply]
Could you say when you're planning on the next release?

**Reply by nicholasgriffintn** [reply]
Working on it, will be some time this week after internal testing.
